msg/sensor_gyro.msg

```diff
@@ -8,6 +8,9 @@ float32 y                 # angular velocity in the FRD board frame Y-axis in ra
 float32 z                 # angular velocity in the FRD board frame Z-axis in rad/s
 
 float32 temperature       # temperature in degrees Celsius
+float32 scale             # sensor resolution scale (LSB/dps)
+
+uint16 dynamic_range      # dynamic range of the gyro (degs/s)
```

Comment:
Again arguably slightly wasteful, but I would keep it consistent with the units of the data above in radians/s (and as a float32).

Reply:
I've been trying to limit bloat for obvious reasons, but psychologically, yes. this is where I want it to be.